src/locale.js

```diff
@@ -59,14 +58,13 @@ const _build = function(stgs) {
   if (!currentLocale) {
     const settings = buildSettings(stgs)
     const lang = settings.locale || defaults.lang
-    const translations = $.extend(
-      true,
```

Comment:
`$.extend(true, obj1, obj2)` is deep, but `Object.assign` is not

Reply:
Yes, you are right. Fixed